docs/DatabaseCatalogAPI.yaml

```diff
@@ -0,0 +1,530 @@
+        location:
+          $ref: "#/components/schemas/FeedLocation"
+        provider:
+          type: string
+          example: London Transit Commission
+        name:
+          type: string
+          example: feed_0
+        urls:
+          $ref: "#/components/schemas/FeedUrls"
+
+    GtfsFeeds:
+      type: array
+      items:
+        $ref: "#/components/schemas/GtfsFeed"
+
+
+    FeedId:
+      type: object
+      properties:
+        feed_id:
+          type: string
+          example: feed_0
+      required:
+        - feed_id
+
+    FeedName:
+      type: string
+      example: Feed 0
+
+    FeedUrls:
+      type: object
+      properties:
+        direct_download:
+          type: string
+          format: url
+          example: http://www.londontransit.ca/gtfsfeed/google_transit.zip
+        authentication_type:
+          type: integer
+          enum:
+            - 0
+            - 1
+            - 2
+        authentication_info_url:
+          type: string
+          format: url
+        api_key_parameter_name:
+          type: string
+          description: The api_key_parameter_name field defines the name of the parameter to pass in the URL to provide the API key. This field is required for authentication_type=1 and authentication_type=2.
+        latest:
+          type: string
+          format: url
+          example: https://storage.googleapis.com/storage/v1/b/mdb-latest/o/us-maine-casco-bay-lines-gtfs-1.zip?alt=media
+
+    FeedLocation:
+      type: object
+      properties:
+        country_code:
+          description: Should be an enum from https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes
+          type: string
+          example: USA
+        subdivision_name:
+          type: string
+          example: California
+        municipality:
+          type: string
+          example: Los Angeles
+        bounding_box:
+          $ref: "#/components/schemas/DatasetGeographicalData"
+
+    FeedStatus:
+      type: string
+      enum:
+        - active
+        - deprecated
+        - inactive
+        - development
+      example: inactive
+
+    Feeds:
+      type: array
+      items:
+        $ref: "#/components/schemas/Feed"
+
+    Dataset:
+      allOf:
+        - $ref: "#/components/schemas/DatasetId"
+        - $ref: "#/components/schemas/DatasetBuilder"
+
+    DatasetBuilder:
+      type: object
+      properties:
+        url:
+          type: string
+          example: http://www.londontransit.ca/gtfsfeed/google_transit.zip
+        type:
+          type: string
+          example: gtfs
+        downloadDate:
+          type: string
+          example: 2020-12-31
+          format: date
+        hash:
+          type: string
+          example: a_long_sha1_hash
+        feed_id:
+          type: string
+          example: feed_0
+        bounding_box:
+          $ref: "#/components/schemas/DatasetGeographicalData"
+
+    DatasetGeographicalData:
```

Comment:
@davidgamez Sorry I missed this comment! Extracted on was only relevant because we generated the bounding box **only on feed creation or feed update**. If we intend to continuously update the bounding box, then it's not needed.